python/standard-library/py-practice-string-manip.md

```diff
@@ -0,0 +1,34 @@
+---
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+  - medium
+
+type: exercise
+
+link: https://www.codewars.com/kata/cat-years-dog-years
+linkType: codewars
+standards:
```

Comment:
nbd now, but we should come back to these files to fix the formatting as it's a bit harder to follow when taking a quick look

actually, we might be able to do this automatically on commits at some point